plotly/tools.py

```diff
@@ -2871,3 +2891,149 @@ def get_candle_decrease(self):
         return (decrease_x, decrease_close, decrease_dif,
                 stick_decrease_y, stick_decrease_x)
 
+class _Dendrogram(FigureFactory):
+    ''' Returns a Dendrogram figure object
+        Example usage:
```

Comment:
It's better to move the example to the `create_dendrogram()` docstring because that's the one that users will see if they call help: help(FigureFactory.create_dendrogram) 
Our idea is that users will only use the create_ methods so the FigureFactory classes are hidden

Then this docstring can be as simple as: Refer to FigureFactory.create_dendrogram() for docstring